src/app/shared/validationMessages.model.ts

```diff
@@ -76,6 +76,18 @@ export const validationMessages = {
     maxlength: 'Descriptor Path is too long. (Max 1000 characters.)',
     pattern: `Invalid Descriptor Path format. Descriptor Path must begin with '/' and end with '*.wdl'.`
   },
+  nflPath: {
+    required: 'This field cannot be empty.',
+    minlength: 'Descriptor Path is too short. (Min. 3 characters.)',
```

Comment:
1. Not 100% sure, but `Min. 3 characters` is not a sentence, so I wouldn't end it with a period.
2. Unless space is short, I would spell out `Minimum`. Max is OK because it's a word on its own, but I would vote against abbreviations.

Point 1 also applies to lines 82, 87, and 88.
Point 2 also applies to line 88.